### PR TITLE
Satellite in anchors part2

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR removes `Satellite` from anchors. It should be rebased + merged after https://github.com/theforeman/foreman-documentation/pull/801.

### CODE REVIEW

**Comment by lzap** [comment]
Part 1 merged, please rebase.

**Comment by lzap** [comment]
After this is merge I will work on https://github.com/theforeman/foreman-documentation/pull/805

**Comment by maximiliankolb** [comment]
Rebased to master.

File: guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc
Comment (melcorr) [inline guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc:1]: what on earth was going on here.

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc
Comment (melcorr) [inline guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:1]: I have to do a git blame to see who did this. 
Perhaps it was me.

File: guides/common/modules/proc_enabling-client-connections-to-satellite.adoc
Comment (melcorr) [inline guides/common/modules/proc_enabling-client-connections-to-satellite.adoc:6]: I remember something about how we really wanted to emphasise how this action needed to happen on the base OS on which you were installing Foreman/Satellite. I think you are obfuscating the point with your change.

**Review by melcorr — CHANGES_REQUESTED** [review]
I have left some comments. 
I want to take a look at the files to understand the logic though I don't expect it to block the PR. 
I just need to know.

File: guides/common/modules/proc_enabling-client-connections-to-satellite.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_enabling-client-connections-to-satellite.adoc:6]: OK, IMHO the `ProjectServer` attribute implies that (in contrast to `Project`). But I've mostly reverted my change. Now, all it removes is `Red Hat Enterprise Linux 7` because Foreman and orcharhino also runs on EL7 clones.

**Comment by maximiliankolb** [comment]
Rebased to master.

**Comment by lzap** [comment]
I have no comments, looks good.

File: guides/common/modules/proc_enabling-client-connections-to-satellite.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_enabling-client-connections-to-satellite.adoc]: Is this ok for you @melcorr ?

**Pull request merged**
